devicon.json

```diff
@@ -925,6 +925,22 @@
         "color": "#1384c8",
         "aliases": []
     },
+    {
+        "name": "dot-net-core",
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```

Comment:
Please create a font version of the icon (see, for example, the [Jupyter](https://github.com/devicons/devicon/tree/master/icons/jupyter) icon).